src/style.css

```diff
@@ -2,0 +1,29 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin: 0;
```

Comment:
Be consistent with your margins. It's better to have only top or bottom margins. Here, `.stars__star` has a `margin: 0;` which is unnecessary since it's already defined in `.stars` and could be overridden by `.stars__star:not(:first-child)`.